packages/fast-form-generator-react/src/form/form-item.array.style.ts

```diff
@@ -18,76 +19,77 @@ const styles: ComponentStyles<FormItemArrayClassNameContract, {}> = {
         ...applyGlobalStyle(),
     },
     formItemArray: {
-        position: "relative",
-        display: "flex",
-        flexDirection: "column",
+        display: "grid",
+        ...applyControlWrapper(),
     },
     formItemArray_control: {
         width: "100%",
+        position: "relative",
         verticalAlign: "middle",
-        marginTop: toPx(12),
     },
     formItemArray_controlAddButton: {
         position: "absolute",
-        right: "0",
-        top: "12px",
+        right: "5px",
+        top: "5px",
         appearance: "none",
         background: "none",
         border: "none",
-        margin: "8px",
         width: "20px",
-        height: "16px",
+        height: "20px",
         zIndex: "1",
         borderRadius: "2px",
         [`&${focusVisible()}`]: {
-            ...insetStrongBoxShadow(colors.pink),
+            ...insetStrongBoxShadow(pink),
             outline: "none",
         },
         "&::before": {
             position: "absolute",
             content: "''",
-            opacity: ".6",
             pointerEvents: "none",
-            width: toPx(16),
-            height: toPx(16),
-            left: toPx(2),
-            top: "0",
-            background: plus,
+            width: "9px",
+            height: "1px",
+            left: "5.5px",
+            top: "9.5px",
+            background: foreground300,
+        },
+        "&::after": {
+            position: "absolute",
+            content: "''",
+            pointerEvents: "none",
+            width: "1px",
+            height: "9px",
+            left: "9.5px",
+            top: "5.5px",
+            background: foreground300,
         },
     },
     formItemArray_controlLabel: {
         ...applyLabelStyle(),
         ...applyControl(),
-        verticalAlign: "middle",
-        lineHeight: "32px",
     },
     formItemArray_existingItemList: {
         ...applyCleanListStyle(),
+        fontSize: "11px",
+    },
+    formItemArray_existingItemList__sorting: {
+        fontSize: "11px",
+        color: foreground300,
+        backgroundColor: `${background100} !important`,
     },
     formItemArray_existingItemListItem: {
         position: "relative",
-        height: "36px",
-        lineHeight: "36px",
-        paddingLeft: "26px",
-        "&::before": {
-            position: "absolute",
-            content: "''",
-            opacity: ".6",
-            pointerEvents: "none",
-            top: toPx(11),
-            width: toPx(16),
-            height: toPx(16),
-            background: lines,
-            left: "0",
-        },
+        paddingLeft: "10px",
+        marginLeft: "-10px",
```

Comment:
It seems like this repeats quite a bit and seems caused by padding being created several layers outside of the actual form itself. We should either remove the padding outside or fix the instances where we are simultaneously adding adding padding and removing margin on the same side of this.